src/app/assinante/assinante.module.ts

```diff
@@ -8,6 +8,8 @@ import { HeaderModule } from '../header/header.module';
 import { FooterModule } from '../footer/footer.module';
 import { HeaderlogModule } from '../headerlog/headerlog.module';
 import { AssinanteComponent } from './component/assinante.component';
+import { AssinanteService } from './service/assinante.service';
+import { ArtistaService } from '../artista/service/artista.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8399665)
```suggestion
import { ArtistaService } from "../artista/service/artista.service";
```